lib/default-notes-template.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+module.exports = ({ agendaIssues, agendaLabel, owner, repo, issue }) => {
+  return `
+# ${issue.title}
+
+## Links
+
+  * **Recording**:
+  * **GitHub Issue**: https://github.com/${owner}/${repo}/issues/${issue.number}
+
+## Present
+
+  *
+
+## Agenda
+
+## Announcements
```

Comment:
Are these flipped? Agenda items are below announcements.

Reply:
On all wgs I participate "Announcements" appear after "Agenda", but I'm happy to flip (it does make more sense to flip)

Reply:
That may be a copy/paste template issue - I'd recommend we flip 😅